hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -96,30 +99,14 @@
     @Option(names = {"-f", "--config"},
             description = "Optional path to a client config XML/YAML file." +
                     " The default is to use config/hazelcast-client.yaml." +
-                    " If you set this option, Jet will ignore the --addresses and --cluster-name options.",
+                    " If you set this option, Jet will ignore the --targets option.",
             order = 0
     )
     private File config;
 
-    @Option(names = {"-a", "--addresses"},
-            split = ",",
-            arity = "1..*",
-            paramLabel = "<hostname>:<port>",
-            description = "Optional comma-separated list of Jet node addresses in the format" +
-                    " <hostname>:<port>, if you want to connect to a cluster other than the" +
-                    " one configured in the configuration file",
-            order = 1
-    )
-    private List<String> addresses;
```

Comment:
I think we still need to keep these, as this is a breaking change. You can mark them as deprecated